var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -53,14 +76,25 @@ def patch(self):
     def manual_install(self, prefix):
         install_tree('include', prefix.include)
         install_tree('lib', prefix.lib)
-        install_tree('documentation', prefix.share + '/libxsmm/doc')
+        doc_path = prefix.share + '/libxsmm/doc'
+        mkdirp(doc_path)
+        for doc_file in glob('documentation/*.md'):  install(doc_file, doc_path)
+        for doc_file in glob('documentation/*.pdf'): install(doc_file, doc_path)
+        install('README.md', doc_path)
+        install('LICENSE', doc_path)
 
     def install(self, spec, prefix):
-        make_args = [
-            'ROW_MAJOR=0',
-            'INDICES_M={0}'.format(' '.join(str(i) for i in range(1, 25))),
-            'INDICES_N={0}'.format(' '.join(str(i) for i in range(1, 25))),
-            'INDICES_K={0}'.format(' '.join(str(i) for i in range(1, 25)))
-        ]
+        # include symbols by default
+        make_args = ['SYM=1']
+
+        # JIT (AVX and later) makes MNK, M, N, or K spec. superfluous
+        #make_args += ['MNK=1 4 5 6 8 9 13 16 17 22 23 24 26 32']
+
+        # TODO: shared variant introduces a BLAS dependency
```

Comment:
The phrase "not stable" means I prefer to contribute such improvement later on. I am just starting with Spack, and I would like to make more incremental contributions rather than rushing something untested into the spec.file (so "unstable" does not refer to blas built by Spack ;-).